admin/class-h5p-plugin-admin.php

```diff
@@ -156,6 +159,79 @@ function add_settings_link($links) {
     return $links;
   }
 
+  /**
+   * Spawn an h5p content via POST request
+   */
+  public function spawn_h5p() {
+    $plugin = H5P_Plugin::get_instance();
+    $action = $_REQUEST['action'];
+    if ($action) {
```

Comment:
Rewrite this using "return early" as a manta

https://betterprogramming.pub/are-early-returns-any-good-eed4b4d03866

In any method which resembles "glue code" or "controller code" like this, I'll happily die on the hill of return early always being the better pattern to follow, every time.